### DISCUSSION TITLE
posts/odoo/Odoo-class-o-row

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#20] [General]
# posts/odoo/Odoo-class-o-row

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/odoo/Odoo-class-o-row.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2022-01-27T23:09:43.146Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.71 Safari/537.36"

class=”o_row”
